spec/models/video_spec.rb

```diff
@@ -2,6 +2,60 @@
 
 describe Video do
   it { should belong_to(:category) }
-  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:title) }
   it { should validate_presence_of(:description) }
+
+  describe "search_by_title" do 
```

Comment:
When you describe the message you should use `#` for instance methods and `.` for class methods - Read more [here](http://betterspecs.org/#describe)

`describe ".search_by_title" do`